lib/src/base-styles/_nx-page.scss

```diff
@@ -25,23 +27,42 @@
 
   .nx-btn-bar {
     grid-area: actions;
-    margin: 0 0 0 $nx-spacing-l;
+    margin: 0 0 0 $nx-spacing-xs;
   }
 }
 
-.nx-page-title__tags {
-  @include container-horizontal;
-
-  grid-area: tags;
-  margin-left: $nx-spacing-md;
+.nx-page-title__headings {
+  display: contents;
+}
 
-  .nx-policy-violation-indicator {
-    margin: 0;
-  }
+.nx-page-title__sub-title {
+  @include nx-truncate-ellipsis;
+  border-left: $nx-border-default;
+  margin: 0;
+  padding-left: $nx-spacing-md;
+  min-width: 150px;
 }
 
 .nx-page-title__description {
   @include container-vertical;
   grid-area: description;
-  margin: $nx-spacing-xs 0 0 0;
+  margin-top: $nx-spacing-xs;
+
+  + .nx-page-title__tags {
+    margin-top: $nx-spacing-md;
+  }
+}
+
+.nx-page-title__tags {
+  @include container-horizontal;
+
+  display: flex;
+  flex-wrap: wrap;
+  grid-area: tags;
+  row-gap: $nx-spacing-xs;
+  margin-top: $nx-spacing-xs;
+
+  > :nth-child(1n) {
```

Comment:
This seems like a weird way of selecting every child.  Is this just to increase specificity? Would `> *, > :first-child, > :last-child` work?

Reply:
> This seems like a weird way of selecting every child. Is this just to increase specificity? Would `> *, > :first-child, > :last-child` work?

Yes it's specificity. `> *` did not work. It's overridden by the specific `.class`'s of the components

Reply:
> Yes it's specificity. > * did not work. It's overridden by the specific .class's of the components

What about what I suggested above?

Reply:
> > Yes it's specificity. > * did not work. It's overridden by the specific .class's of the components
> 
> What about what I suggested above?

It works. Why exactly it is better than what I have?

Reply:
> > > Yes it's specificity. > * did not work. It's overridden by the specific .class's of the components
> > 
> > 
> > What about what I suggested above?
> 
> It works. Why exactly it is better than what I have?

I accidentally merged it: https://github.com/sonatype/sonatype-react-shared-components/pull/363/commits/b4660408e77ea0f60e58dc9f88587b385780dbd3

Reply:
> Why exactly it is better than what I have?

I think it's better in two ways
1. The `nth-child` expression was confusing.  I looked at it, said to myself, "wait, what children does that apply to, isn't it all of them?" and then had to go into the browser to confirm, and then ponder why it was written that way.
2. What I suggested is more clear about what other specificities it has to beat (other `:first-child` and `:last-child` selectors)